### DISCUSSION TITLE
WindTerm支持应用传递路径吗？

### DISCUSSION THREAD

**Discussion by lanjackg2003** [kingToolbox/WindTerm#1348] [Q&A]
我传递一个路径参数给`WindTerm.exe`,但发现WindTerm界面中的shell界面部分,打开的路径仍然是我的用户名路径
![图片](https://user-images.githubusercontent.com/20794381/201559903-e64de89a-e636-4412-9e5e-f72fdee15dc2.png)

![图片](https://user-images.githubusercontent.com/20794381/201559858-6abeda4c-144c-43c6-9fa7-9a975bcdc45b.png)

那WindTerm可以支持路径参数加载吗？
可以的话,要如何传参?

**Comment by 78zhanghao87** [comment]
有类似的问题
